tests/test.hc.serve.sample.test

```diff
@@ -0,0 +1,4 @@
+docker run  -e LOCAL_USER_ID=`id -u $USER` -i devcore /bin/sh -c tests/test.hc.serve.sample.test.inContainer | tee test.hc.serve.sample.test.output
+grep -q "Starting with UID : 1002" test.hc.serve.sample.test.output || exit 1
+rm test.hc.serve.sample.test.output
+echo "passed test served hc as as user with matching user id"
```

Comment:
I'm not sure I get the idea of creating a user in the container that matches the outside UID. This the test run user as distinct from the install owner (holochain)? I'm not sure there is any additional isolation gained.